src/buildercore/utils.py

```diff
@@ -36,6 +36,19 @@ def dictfilter(func, ddict):
 def dictmap(fn, ddict):
     return {key: fn(key, val) for key, val in ddict.items()}
 
+def nested_dictmap(fn, ddict):
```

Comment:
where is this function used? I can't find a call to it apart from tests

Reply:
it was part of the project vars interpolation that I reverted. I opted to keep it because it was useful. I plan to extract all my little utilities into their own module so I don't keep duplicating them across projects. Will open a ticket and talk to Sean about it.